topics-project/terraform/provider.tf

```diff
@@ -0,0 +1,3 @@
+provider "aws" {
+    region = "us-east-1"
```

Comment:
It's a good idea to deploy to in one region when unless you explicitly want to target two regions for redundancy.